frontend/src/Components/RoadDetails/ImageClick.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+interface ImageClickProps {
+  imageUrl: string;
+  onClose: () => void;
+  onNavigate: (direction: number) => void;
+  currentImageIndex: number;
+  totalImages: number;
+}
+
+const ImageClick: React.FC<ImageClickProps> = ({ imageUrl, onClose }) => {
```

Comment:
```suggestion
/**
 * ImageClick make images clickable and open a popup to see image closer
 */
const ImageClick: React.FC<ImageClickProps> = ({ imageUrl, onClose }) => {
```
Add description for the documentation

Reply:
Also, according to the [System specification](https://docs.google.com/document/d/1wuZVGP4DW73j9L2VWPJ28040FBKVGpg1_IRyC1MvOjI/edit#bookmark=id.safk542bqoa6), the component name should be ZoomableImage